### DISCUSSION TITLE
Setting up CrowdSec Repository...‼ ERROR 1@63 Unknown failure occured.

### DISCUSSION THREAD

**Discussion by circa1741** [tteck/Proxmox#247] [General]
I was trying out the instructions in:
`https://github.com/tteck/Proxmox/discussions/50#discussioncomment-2834124`

To create a new Proxmox Debian 11 (curl. sudo) LXC, I ran the following in the Proxmox Shell:
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/debian-v3.sh)"`

Then I ran in the existing LXC console:
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/crowdsec-v3.sh)"`

I received the message:
`Setting up CrowdSec Repository...‼ ERROR 1@63 Unknown failure occured.`

![image](https://user-images.githubusercontent.com/8772586/170836139-fd0cc275-0884-44c1-97ea-dc260f296133.png)

**Comment by tteck** [comment]
After the Debian LXC creation, run `apt update` in the LXC console then install CrowdSec

**Reply by circa1741** [reply]
rock on! thanks.

![image](https://user-images.githubusercontent.com/8772586/170838050-6d8c2609-f338-4fef-812d-ba8a7ec2c45a.png)

**Reply by tteck** [reply]
https://app.crowdsec.net/

**Reply by circa1741** [reply]
nice! thanks.

**Reply by Koliera** [reply]
Aye, followed everything as per above but getting the following:
 - Installing CrowdSec...‼ ERROR 100@68 Unknown failure occured.

Would love to get CrowdSec going. Always thought it might be interesting to set up.

**Reply by tteck** [reply]
@Koliera I'm now getting the same error. Removing until  CrowdSec/packagecloud resolves the issue

**Reply by Koliera** [reply]
> @Koliera I'm now getting the same error. Removing until CrowdSec/packagecloud resolves the issue

Weird, I restarted the container a couple times and it went through yesterday. Seems very hit/miss. But yes, I'd love more integration of this. For example it being attached together with the NPM and maybe Authelia. That seems like a good pack to have security wise. One thing to remember is that NPM isn't Nginx per se, it's Resty so the scenarios used for CrowdSec have to be appropriately selected.

**Comment by JOduMonT** [comment]
I just fixed the error ;)
https://github.com/tteck/Proxmox/pull/256
![image](https://user-images.githubusercontent.com/5204724/171751126-471f65d3-5755-4c7e-b2a4-5263f81fa96d.png)

**Reply by tteck** [reply]
the `https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.deb.sh` script installs gnupg on line 23

**Reply by tteck** [reply]
It won't hurt anything to add it to dependencies
I'll add it

**Reply by tteck** [reply]
I've noticed that a newly created LXC needs a reboot before adding CrowdSec. 🤷🏻‍♂️

**Comment by Val-Prok** [comment]
` - Setting up CrowdSec Repository...‼ ERROR 127@66 Unknown failure occured.`
 
![image](https://user-images.githubusercontent.com/71496555/192035397-b18ddde5-1473-44df-95a4-975287957c3a.png)

:^(

**Reply by tteck** [reply]
You're running it in the Proxmox Shell?
Just tested
```
   _____                      _  _____           
  / ____|                    | |/ ____|          
 | |     _ __ _____      ____| | (___   ___  ___ 
 | |    |  __/ _ \ \ /\ / / _  |\___ \ / _ \/ __|
 | |____| | | (_) \ V  V / (_| |____) |  __/ (__ 
  \_____|_|  \___/ \_/\_/ \__ _|_____/ \___|\___|

 ✓ Setup CrowdSec Repository
 ✓ Installed CrowdSec on debian
 ✓ Installed CrowdSec Common Bouncer
 ✓ Completed Successfully!

root@debian:~# 
```

**Reply by Val-Prok** [reply]
Yes, in Proxmox Shell

![image](https://user-images.githubusercontent.com/71496555/192041848-73d41f2d-1dbc-4fdf-92da-8eb046bd9949.png)

**Reply by tteck** [reply]
Yeah I seen that, but why?
